modules/tracktion_engine/playback/graph/tracktion_MidiInputDeviceNode.cpp

```diff
@@ -138,7 +138,7 @@ void MidiInputDeviceNode::processSection (ProcessContext& pc, juce::Range<int64_
     if (timeNow > lastReadTime + maxExpectedMsPerBuffer)
     {
         //jassertfalse
-        numMessages = 0;
+        //numMessages = 0;
```

Comment:
I'm not sure why this is in this PR? Can it be removed so we can consider it separately?